src/lib/assets/store.ts

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { PeprStore, modelToGroupVersionKind } from "../k8s";
+import { CustomResourceDefinition } from "../k8s/upstream";
+
+export const { group, version, kind } = modelToGroupVersionKind(PeprStore.name);
+export const singular = kind.toLocaleLowerCase();
+export const plural = `${singular}s`;
+export const name = `${plural}.${group}`;
+
+export const peprStoreCRD: CustomResourceDefinition = {
+  apiVersion: "apiextensions.k8s.io/v1",
+  kind: "CustomResourceDefinition",
+  metadata: {
+    name,
+  },
+  spec: {
+    group,
+    versions: [
+      {
+        // typescript doesn't know this is really already set, which is kind of annoying
+        name: version || "v1",
+        served: true,
+        storage: true,
+        schema: {
+          openAPIV3Schema: {
+            type: "object",
+            properties: {
+              data: {
+                type: "object",
+                additionalProperties: {
+                  type: "string",
+                },
+              },
+            },
+          },
+        },
+      },
+    ],
+    scope: "Namespaced",
+    names: {
```

Comment:
I actually chose not to in order to avoid possible conflicts, it would be a minor convenience, but decided it wasn't worth the conflict risk. Thoughts?

Reply:
sorry for the late reply, possible conflicts with other `ps` resources?

I'm okay either way. At Solo, Gloo Edge had shortName for `vs` which would clobber Istio's `vs`, the work around was fairly easy to remove the CRD. 

As kubernetes tool i like the shortname because it feels more native and I use shortnames constantly anyway, `k get po,svc,deploy,cm,etc`